packages/client/archive/libp2p/index.ts

```diff
@@ -8,9 +8,9 @@ import { Config } from '../src/config'
 import { LevelDB } from '../src/execution/level'
 import { parseMultiaddrs } from '../src/util'
 
-import { getLogger } from './logging'
+import { getLogger } from './logging.js'
 // Blockchain
-export * from '../src/blockchain/chain'
+export * from '../src/blockchain/chain/index.js'
```

Comment:
Why are we adding that `.js` suffix in some places but not all?

Reply:
I shouldn't have addit it all in this `archive` folder. Just find/replace errors.  We can safely disregard anything in this folder.